docs/source/index.rst

```diff
@@ -14,24 +14,31 @@ It provides a comprehensive overview on what you can do with packages that are p
 If you are new to **signac**, the best place to start is to read the :ref:`introduction` and the :ref:`tutorial`.
 
 
-Contents
---------
-
 .. toctree::
    :maxdepth: 2
+   :caption: Getting started
 
    intro
    installation
    quickstart
    tutorial
    reference
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Extra
```

Comment:
I dislike the name "Extra." I'm not sure if this categorization makes sense. Maybe "API and Usage"?

Reply:
I've changed it to "User Guide".